demisto_sdk/commands/common/tools.py

```diff
@@ -830,6 +830,32 @@ def get_file(
         return {}
 
 
+def get_file_or_remote(file_path: Path, clear_cache=False):
+    content_path = get_content_path()
+    relative_file_path = None
+    if file_path.is_absolute():
+        absolute_file_path = file_path
+        try:
+            relative_file_path = file_path.relative_to(content_path)
+        except ValueError:
+            logger.debug(f"{file_path} is not a subpath of {content_path}")
```

Comment:
then what happens?

Reply:
If the file exists locally, nothing.
If it doesn't it will raise an exception later.

Reply:
then let's add this info to the log, to make it easy for users and debuggers

Reply:
done